org.kde.akregator.json

```
@@ -0,0 +1,495 @@
+{
+    "id": "org.kde.akregator",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-21.08",
+    "command": "akregator",
```

```suggestion
    "command": "akregator",
    "rename-icon": "akregator",
```

There were complaints in the build output, not sure how it passed.